docs/rules/no-test-import-export.md

```diff
@@ -0,0 +1,76 @@
+# No importing of test files
+
+**TL;DR** Do not import "-test.js" file in a test file. This will cause the module and tests from the imported file to be executed again. In the same vain, no exports within a test file.
```

Comment:
Do not import from a test file (a file ending in "-test.js") in another test file. Doing so will cause the module and tests from the imported file to be executed again. Similarly, test files should not have any exports.